src/pages/SubmissionDetails.js

```diff
@@ -78,24 +77,11 @@ export const SubmissionDetails = ({ submissionId }) => {
       }
     }
 
-    const loadMethodsData = async () => {
-      try {
-        const methodsResponse = await Ajax(signal).Cbas.methods.get()
-        const allMethods = methodsResponse.methods
-        setMethodsData(allMethods)
-      } catch (error) {
-        notify('error', 'Error loading methods data', { detail: await (error instanceof Response ? error.text() : error) })
-      }
-    }
-
```

Comment:
I would leave this stuff in... mainly because I will be needing it in the next PR (even if it's not 100% needed right now)